services/addons/images/firmware_manager/yunex_upgrader.py

```diff
@@ -0,0 +1,117 @@
+import upgrader
+import json
+import logging
+import os
+import subprocess
+import sys
+import tarfile
+import time
+
+class YunexUpgrader( upgrader.UpgraderAbstractClass ):
+  def __init__(self, upgrade_info):
+    super().__init__(upgrade_info)
+
+  def run_xfer_upgrade(self, file_name):
+    xfer_command = ["java", "-jar", f"/home/tools/xfer_yunex.jar", "-upload", file_name, f"{self.rsu_ip}:3600"]

```

Comment:
Might be good to have the rsu port as an env variable just in case this changes.

Reply:
Naw, this is a Yunex standard. We don't want the user to have too much control.